doc/debian/jitsi-meet/jitsi-meet.conf

```diff
@@ -11,9 +11,8 @@ stream {
     }
     # since 1.13.10
     map $ssl_preread_alpn_protocols $upstream {
-        "h2"            web;
-        "http/1.1"      web;
-        "h2,http/1.1"   web;
+        ~\bh2\b         web;
+        ~\bhttp/1.1\b   web;
```

Comment:
Small nitpick, but you should probably escape the dot.

Reply:
Yes, right! Revising as suggested.

Reply:
On further thought, there's no reason why we shouldn't proxy HTTP 1.2 (or any future version of HTTP 1.x) to the web server too.